substrate/frame/contracts/src/lib.rs

```diff
@@ -859,19 +857,26 @@ pub mod pallet {
 			use migration::MigrateResult::*;
 			ensure_signed(origin)?;
 
-			let weight_limit = weight_limit.saturating_add(T::WeightInfo::migrate());
-			let (result, weight) = Migration::<T>::migrate(weight_limit);
+			let limit = weight_limit.saturating_add(T::WeightInfo::migrate());
```

Comment:
nit: Let's just shadow the unused variable. Alternativly we can just rename the parameter.

```suggestion
			let weight_limit = weight_limit.saturating_add(T::WeightInfo::migrate());
```